setup.rst

```diff
@@ -39,8 +39,8 @@ requirements. Open your console terminal and run this command:
 
 .. note::
 
-    The Symfony binary is developed internally at Symfony. If you want to
-    report a bug or suggest a new feature, please create an issue on
+    The Symfony CLI source is available at the `Symfony CLI repository`_. If
```

Comment:
```suggestion
    The Symfony CLI sources are available at the `Symfony CLI organization repository`_. If
```

what about this?

Reply:
I agree with the link anchor, but the source is singular, not plural.